5/alpine/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # https://docs.ghost.org/faq/node-versions/
 # https://github.com/nodejs/Release (looking for "LTS")
-FROM node:18-alpine3.19
+FROM node:18-alpine3.20
```

Comment:
Following those two links, should we bump the node version to 20?  It is the "recommended" version from Ghost and 18 will be end of life at the end of April 2025. So, something to think about before April.

Reply:
https://github.com/docker-library/ghost/pull/414 :sob:

Reply:
(It's probably worth trying a variant of that which is _just_ the version bump, not touching `libvips`)